### ISSUE TITLE
[Request] Sleek circular slider

### ISSUE THREAD

**Issue by matthewfx** [flutterarsenal/FlutterArsenal#73]
github: https://github.com/matthewfx/sleek_circular_slider
tag: Slider 
excerpt: The highly customizable circular slider/progress bar  
email-id: mat@nuckowski.com 
teaser:  
![sleek_circular_slider06](https://user-images.githubusercontent.com/3390107/66375630-577ea180-ea0a-11e9-9b49-daa970b17906.GIF)

**Comment by flutter-arsenal** [comment]
Thank you @matthewfx for submitting a new weapon for the __FlutterArsenal__.
The github repository: [_matthewfx/sleek_circular_slider_](https://github.com/matthewfx/sleek_circular_slider) is now awaiting approval from __admins__.

Kudos to you for contributing! cc @all-contributors please add @matthewfx for content and ideas.

**Comment by karx** [comment]
@IAmSarthakVerma Please review and update

**Comment by flutter-arsenal** [comment]
Thank you @matthewfx for submitting a new weapon for the __FlutterArsenal__.
The github repository: [_matthewfx/sleek_circular_slider_](https://github.com/matthewfx/sleek_circular_slider) is now awaiting approval from __admins__.

Kudos to you for contributing! cc @all-contributors please add @karx for content and ideas.

**Comment by flutter-arsenal** [comment]
Congratulations! @matthewfx your request is now approved and live on [__FlutterArsenal__](https://flutterarsenal.com).

Ping! @matthewfx. Your project is now listed.
Please help and support us in maintaining the biggest arsenal of Flutter weapons.
[![Tip Me via PayPal](https://img.shields.io/badge/PayPal-tip%20me-green.svg?logo=paypal)](https://www.paypal.me/karx01)

**Comment by matthewfx** [comment]
Guys thanks for adding my widget but I have a question. What has happened to the gif I used for the teaser? Why it is not displayed on the website?

**Comment by karx** [comment]
I have created a [Flutter Arsenal webhook](https://github.com/karx/FlutterArsenal-issuehook), which is service, triggered by issue activities of this repository.

On project approve label - it automatically pulls data from the issue you created and pushes a new commit with the project listing to this repository, which triggers a new build.

The rule matching for image expected a direct link. Need to update that. Just a regex update is needed in the getTeaserRx function here: [issue-handler.js #L407](https://github.com/karx/FlutterArsenal-issuehook/blob/eca382005eeb3656b7d9a83398cdfeca500d836f/issue-handler.js#L407)

@matthewfx 
Fixed it manually for now, and committed. Should be fine now.
